src/app/ColorMap.ts

```diff
@@ -14,76 +11,76 @@ export class ColorMap {
    * Assumes h, s, and l are contained in the set [0, 1] and
    * returns r, g, and b in the set [0, 1].
    *
-   * @param   Number  h       The hue
-   * @param   Number  s       The saturation
-   * @param   Number  l       The lightness
+   * @param     h       The hue
+   * @param     s       The saturation
+   * @param     l       The lightness
    * @return  Array           The RGB representation
    */
-  hslToRgb (h : number, s : number, l : number){
-    var r, g, b;
+  static hslToRgb (h: number, s: number, l: number) {
```

Comment:
This whole file is broken. All of the `static` stuff broke it. When I play around and fix one static, the others break. I think you should omit this file from any changes for now.